src/containers/App/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Header from '../../components/Header';
+import Board from '../Board';
+import cardReducer from '../../reducers';
+import './App.css';
+
+class App extends Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      headerTitle : `KANBAN`,
+    }
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Header title={ this.state.headerTitle } />
+        {/* the Board component should have a cards prop referencing the array of cards that we get from the reducer. */}
+        <Board cards={ cardReducer } />
```

Comment:
Don't think you need to pass in the reducer to your component in this way. That's what the redux `connect` method will do on your `Board` component.